docker/Dockerfile.hipcc

```diff
@@ -100,7 +100,7 @@ RUN ARBORX_VERSION=4834bff44c23c9510c6ed93366638dcdf85ab217 && \
       -D CMAKE_INSTALL_PREFIX=${ARBORX_DIR} \
       -D CMAKE_BUILD_TYPE=Debug \
       -D CMAKE_CXX_COMPILER=hipcc \
-      -D CMAKE_CXX_FLAGS=-amdgpu-target=gfx906 \
+      -D CMAKE_CXX_FLAGS=-amdgpu-target=gfx908 \
```

Comment:
Wait why do we add flags here?

Reply:
ping

Reply:
Looks like we've had that since we added TPL to HIP builds. All the jenkins builds need version/option updates - are you okay if I do that separately?

Reply:
Sure.  You might want (whenever you handle this) to pass `-D GPU_TARGETS=gfx908`.
This is intended for rocThrust, making sure that it does not make you compile for all AMD GPU architectures that cmake knows about.